src/App.tsx

```diff
@@ -1,61 +1,109 @@
+import React from 'react';
 import './App.scss';
-
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { getUserById, todos } from './components/todosStaticList';
+import { TodoList } from './components/TodoList';
+import users from './api/users';
+import { useState } from 'react';
+import { Todo } from './types';
 
 export const App = () => {
+  const [currTodos, setCurrTodos] = useState<Todo[]>(todos);
+  const [titleValue, setTitleValue] = useState('');
+  const [selectUser, setSelectUser] = useState(0);
+  const [inputError, setInputError] = useState(false);
+  const [selectError, setSelectError] = useState(false);
+
+  const reset = () => {
+    setTitleValue('');
+    setSelectUser(0);
```

Comment:
Here you can use a variable like this: const `DEFAULT_USER_ID = 0;` to avoid magic values.
```suggestion
  const [selectUser, setSelectUser] = useState(DEFAULT_USER_ID);
  const [inputError, setInputError] = useState(false);
  const [selectError, setSelectError] = useState(false);

  const reset = () => {
    setTitleValue('');
    setSelectUser(DEFAULT_USER_ID);
```